src/client/components/collection/Badges.js

```diff
@@ -25,8 +26,27 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
-function Badges({ views, likes }) {
+function Badges({ views, likes, name }) {
   const classes = useStyles();
+  const [pageLocation, setPageLocation] = useState(null);
+
+  useEffect(() => {
+    setPageLocation(window?.location.href);
+  });
+
```

Comment:
How's it looking on Twitter and FB? Any previews? It would be good to also include the `og` tags so that we get the right preview. @hasanagh and @hagoptaminian  can help as they did it for the other repos.